site/docs/versions.mdx

```diff
@@ -0,0 +1,95 @@
+---
+id: versions
+title: Bandwidth Documentation Versions
+slug: /versions
+description: Bandwidth's Developer Docs Versions
+sidebar_label: Versions
+hide_table_of_contents: true
+keywords:
+  - bandwidth
+  - docs
+  - documentation
+  - api
+  - versioning
+  - LTS
+hide_title: false
+image: '@site/static/img/bw-icon.svg'
+---
+
+import LTSVersions from '@site/src/components/LTSVersions.js';
+
+export const versions = [
+    'v2022.10.17'
+]
+
+# Bandwidth Documentation Versions
+
+Bandwidth maintains both current and LTS versions of our docsite and all of its contents.
+What you will find here (dev.bandwidth.com), will always be the most up-to-date version of our docsite.
+If you need to access an old version of our site, you can check the [LTS Versions](#lts-versions) table below.
+Our versions use date based versioning in the format `vYYYY.MM.DD`.
+
+### LTS Versions
+
+This table contains all of our current LTS versions of the doc site, which are snapshots of the site and its contents taken on the 1st and 16th of each month.
+We maintain up to 12 LTS versions, or 6 months of old content. After 6 months, the LTS versions can still be accessed via Github, but will not be hosted online.
+
+<LTSVersions versions={versions}/>
+
+:::info
+If you need to access a version of the site not listed as an LTS Version, either because the version was not published as an LTS release,
+or because the LTS version is outside of our 6 month support window, please see the [Other Versions](#other-versions) sections below.
+:::
+
+### Other Versions
+
+Even though we only release an LTS version of the site twice a month, we release changes to the site almost daily.
```

Comment:
```suggestion
Though we only release an LTS version of the site twice a month, we may occasionally release changes to the site more frequently than that.
```

Since ideally our site shouldn't be changing daily, we probably shouldn't make it sound like things need to be fixed or updated every day lol. @dyeam-bw What do you think?

Reply:
```suggestion
Even though we only release an LTS version of the site twice a month, the site can have changes published as frequently as once a day. This is possible during periods when multiple domain teams are making documentation changes.
```
Here's another one. Less incriminating. What do you think?

Reply:
https://www.youtube.com/watch?v=EyKF_cOuwno